upgrade/1.3/scripts/sls/README.SLS_Upgrade.md

```diff
@@ -48,15 +48,15 @@ During the update of SLS, at a minimum, answers to the following questions must
     * Example 1: Upgrade, using the CHN as the system default route (will by default output to `migrated_sls_file.json`).
 
         ```bash
-        ncn# ./sls_updater_csm_1.2.py --sls-input-file sls_input_file.json \
+        ncn# ./sls_updater_csm_1.3.py --sls-input-file sls_input_file.json \
```

Comment:
The actual script file is not being renamed with this PR, so this command won't exist